zerver/tests/test_push_notifications.py

```diff
@@ -969,6 +987,18 @@ def test_non_bouncer_push_remove(self) -> None:
                     'zulip_message_id': message.id,
                 },
                 {'priority': 'normal'})
+            mock_send_apple.assert_called_with(
+                self.user_profile.id,
+                apple_devices,
+                {'badge': 0,
+                 'custom': {
+                     'zulip': {
+                         'gcm': True,
+                         'event': 'remove',
+                         'zulip_message_ids': str(message.id),
+                         'zulip_message_id': message.id
```

Comment:
Same testing setup comments as above; remove the `get_base_payload` mocking to make this test work properly; right now it's verifying wrong output when the code is actually correct.